### DISCUSSION TITLE
New postgres extension

### DISCUSSION THREAD

**Discussion by 0xDebabrata** [supabase/supabase#7236] [Feature Requests] [answered]
I was looking at the `pgvector` extension recently since I want to play with vector embeddings using a postgres database. However I realised that it is currently not possible to install extensions that are not listed in the dashboard. Is it possible to add a new feature that will allow users to install any postgres extension? Or I would really appreciate it if Supabase starts supporting `pgvector` soon!

Here's a meme
![Screenshot 2022-06-09 at 7 48 28 PM](https://user-images.githubusercontent.com/37393381/172869342-12077f8d-9a20-4dbd-ae97-b64d3a33cd2f.png)

**Comment by GaryAustin1** [comment]
This is the "official" extension request thread.  pgvector is in there, you might want to comment or at least up vote.
https://github.com/supabase/supabase/discussions/679

**Reply by 0xDebabrata** [reply]
Ah thank you very much! I was wondering why I couldn't find a proper extension request thread. Should have searched more thoroughly...

**Comment by mattaylor** [comment]
Any thoughts on when / if this extension might be supported?

**Comment by saccodd** [comment]
I have seen a couple of blogposts where they mention that the extension is already available (See here: https://supabase.com/docs/guides/database/extensions/pgvector), but when I try enabling it in SQL I get the following error:

`Failed to run sql query: extension "vector" is not available`

Also, the extension is not available in the database tab

**Reply by saccodd** [reply]
I was working on an old instance. If you create a new database you will see the extension

**Reply by louis030195** [reply]
you can set it up as explained here https://docs.embedbase.xyz/self-hosting/setting-up-supabase

**Comment by inian** [comment, marked as answer]
PgVector is supported - https://supabase.com/docs/guides/database/extensions/pgvector
